src/deployment/cd.md

```diff
@@ -327,4 +327,4 @@ information.
 [Xcode Cloud workflow]: https://developer.apple.com/documentation/xcode/xcode-cloud-workflow-reference
 [custom build scripts]: https://developer.apple.com/documentation/xcode/writing-custom-build-scripts
 [predefined environment variables]: https://developer.apple.com/documentation/xcode/environment-variable-reference
-[Setting the next build number for Xcode Cloud builds]: https://developer.apple.com/documentation/xcode/setting-the-next-build-number-for-xcode-cloud-builds#Set-the-next-build-number-to-a-custom-value
\ No newline at end of file
+[Setting the next build number for Xcode Cloud builds]: https://developer.apple.com/documentation/xcode/setting-the-next-build-number-for-xcode-cloud-builds#Set-the-next-build-number-to-a-custom-value
```

Comment:
What is the difference between these two lines? I'm guessing addition or removal of a trailing newline at EOF?

Reply:
Yeah, it's a trailing EOF I’m afraid (did it via the GH online editor). I can clean it up if needed.

Reply:
Ahh, cool. I'm happy for this file to have a trailing new line. =)